components/sources/ConnectWithFacebook.tsx

```diff
@@ -16,15 +18,41 @@ const FacebookLoginButton: React.FC<FacebookLoginButtonProps> = ({
   isConnected,
   onFacebookLogin,
 }) => {
-  const responseFacebook = (response: any) => {
+  const { currentOrganization } = useOrganizationContext();
+
+  const responseFacebook = async (response: any) => {
+    const user = await AuthApp.currentUser?.getIdToken();
     if (response.accessToken) {
       HttpService.login({
         accessToken: response.accessToken,
         userId: response.userID,
         name: response.name,
         id: response.userID,
       })
-        .then((r) => {
+        .then(async (r) => {
+          const responseToSend = r.data.user;
+
+          await axios.post(
+            "https://be.crowdy.chat/api/v1/sources/facebook",
```

Comment:
Let's keep baseUrl in a constant file so that in the future if required to change it, we can change it from one place.